docs/references/config/genesis.json.md

```diff
@@ -0,0 +1,217 @@
+|:--------------------|:---------------------------------------------------------------------------------------|
+| **Possible values** | &gt;= `"0"`; &lt;= [consensus_params.block.max_bytes](#consensus_paramsblockmax_bytes) |
+
+## consensus_params.validator.pub_key_types
+List of public key types accepted for consensus validation.
+
+| Value type                           | array of string |
+|:-------------------------------------|:----------------|
+| **Possible values of array strings** | `"ed25519"`     |
+|                                      | `"secp256k1"`   |
+
+Note that the restriction to the specified types is only checked when the network is already running and there are
+changes in the validator set. This means that in the genesis file's [validators](#validators) section, you can define
+both types of consensus keys without restriction.
+
+## consensus_params.version.app
+Consensus parameter set version.
+
+| Value type                           | string |
+|:-------------------------------------|:-------|
+| **Possible values of array strings** | `"0"`  |
+
+This is used by the ABCI application on the chain. It can also be updated on chain by the ABCI application.
+
+## consensus_params.abci.vote_extensions_enable_height
+If you enable vote extensions in the genesis file, you have to decide at which height are the extensions enabled.
+
+> Note: If a value `0` is used, it means the extension will not be enabled. In order for it to be enables, a
+> value greater than `0` needs to be specified.
+
+| Value type                           | string      |
+|:-------------------------------------|:------------|
+| **Possible values of array strings** | &gt;= `"0"` |
+
+This can be enabled by governance if it is not enabled in the genesis file. Once active, vote extensions cannot be
+disabled. (Vote extensions can be still disabled by governance, if they are supposed to be enabled at a future height.)
+
+## validators
+List of initial validators for consensus.
+
+| Value type                              | array of objects |                                                                                         |
+|:----------------------------------------|:-----------------|-----------------------------------------------------------------------------------------|
+| **Mandatory keys of each array object** | address          | See [address](priv_validator_key.json.md#address) in priv_validator_key.json            |
+|                                         | pub_key          | See [pub_key.type](priv_validator_key.json.md#pub_keytype)                              |
+|                                         |                  | and [pub_key.value](priv_validator_key.json.md#pub_keyvalue) in priv_validator_key.json |
+|                                         | power            | &gt;= `"0"`                                                                             |
```

Comment:
I think we can't set power 0 during genesis, but not 100% sure

Reply:
No, we can't: https://github.com/cometbft/cometbft/blob/cason/2713-redundant-prevote/types/genesis.go#L91

Reply:
Thanks for confirming. Changing the text.